requirements.txt

```diff
@@ -12,6 +12,7 @@ ujson
 autoflake
 black==20.8b1
 databases[sqlite]
+flake8
```

Comment:
Not doing flake8-bugbear or flake8-pie for now. They're not passing and this PR is big enough already.